src/judoscale/django/apps.py

```diff
@@ -27,14 +27,12 @@ def install_middleware(self):
             logger.info("Unable to install middleware")
             return False
 
-        logger.info("Installing middleware")
-
```

Comment:
This should log only if the middleware is missing, so after the expression: `if judoscale_middleware not in settings.MIDDLEWARE`